src/services/ratingService.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-plusplus */
+
+import { updateOrCreate } from "./userService";
+import { Rating, Trips} from '../database/models';
+
+const getRatingArray = (array)=>{    // function to filter the array of rating query to get only ratings
```

Comment:
Leave your service for model interactions and put the rest in controller please

Reply:
I did it you can review it again